ci/provision-acs-interface.sh

```diff
@@ -8,10 +8,12 @@ sudo pip3 install pipenv
 cd /vagrant
 pipenv install --ignore-pipfile 2> /dev/null
 mkdir -p data
+cp ./examples/.env .
 
 container=$(docker ps -f name=minio -aq)
 if [ -z $container ]; then (
-  pipenv run examples/minio.sh
+  sed -i 's/\r$//' examples/minio.sh
+  pipenv run bash examples/minio.sh
```

Comment:
Maybe [this SO answer](https://stackoverflow.com/a/35986153) has a more elegant fix? Ultimately, it's a bug that files end up borked inside the VM.

Reply:
Thanks! It looks a lot better now!